addons/laser/functions/fnc_toggleMarker.sqf

```diff
@@ -0,0 +1,63 @@
+#include "script_component.hpp"
+/*
+ * Author: LorenLuke
+ * Toggles the marker laser mode for any laser-designating vehicle;
+ *
+ * Arguments:
+ * 0: Vehicle <OBJECT>
+ *
+ * Return Value:
+ * Nothing
+ *
+ * Example:
+ * [vehicle player] call ace_laser_fnc_toggleMarker
+ *
+ * Public: No
+ */
+ 
+
+params ["_vehicle"];
+
+_hasMarker = _vehicle getVariable [QGVAR(hasMarkerLaser), false];
+if (! _hasMarker) exitWith {};
+
+private _enabled = _vehicle getVariable [QGVAR(laserMarkerOn), false];
+_vehicle setVariable [QGVAR(laserMarkerOn), !_enabled];
+
+private _markerMessage = if(_vehicle getVariable [QGVAR(laserMarkerOn), false]) then {localize LSTRING(LaserMarkOn)} else {localize LSTRING(LaserMarkOff)};
+[_markerMessage] call EFUNC(common,displayTextStructured);
+
+GVAR(TrackerpfID) = [{
+    params ["_args", "_pfID"];
+    _args params ["_vehicle"];
+
+    private _lightpoints = _vehicle getVariable [QGVAR(markerlightpoints), []];
+    {
+        deleteVehicle _x;
+    } forEach _lightpoints;
+    _vehicle setVariable [QGVAR(markerlightpoints), []];
+
+    if ( !(_vehicle getVariable [QGVAR(laserMarkerOn), false]) || !(alive _vehicle) ) exitWith {
```

Comment:
```suggestion
    if (!(_vehicle getVariable [QGVAR(laserMarkerOn), false]) || !(alive _vehicle)) exitWith {
```

Reply:
This can be simplified: `!a || !b` == `!(a && b)`